releasenotes/releasenotes.rst

```diff
@@ -26,13 +26,66 @@ The default FHIR version of the `public Firely Server endpoint <https://server.f
+    
+    Here are some guidelines:
+
+   * We tested it on a MongoDb collection with a size of 500GB. The upgrade script took around 24 hours to complete on a fairly powerful machine.
+   * As always, make sure you have a backup of your database that has been tried and tested before you begin the upgrade.
+   * Please make sure that Firely Server is shutdown before you execute the script.
+   * If you encounter problems running the script, or need any assistance, please :ref:`vonk-contact`.
+
+Fix
+^^^
+#. Fixed an issue where a "/" was missing in the fullUrl of a "search" bundle in case a information model mapping with mode "Path" was used.
+#. Fixed an issue where a new resource id was not created when POST was used in a batch or transaction bundle and a resource id was already provided.
+#. An invalid system URI was provided by default in AuditEvent.source.observer.identifier.
+#. Adjusted the implementation of conditional create to match the description in https://jira.hl7.org/browse/FHIR-31965.
+#. Money.currency was not indexed correctly in FHIR R4. Please :ref:`contact us<vonk-contact>` if you are using the SearchParameters "price-override" on ChargeItem or "totalgross" / "totalnet" on Invoice. A migration for these fields will be provided upon request. Otherwise, please re-index these SearchParameters.
+#. Fixed an issue where bundles with conformance claims in meta.profile would have been validated against the profile claims even if the validation level was only set to "Core".
+#. Validating a resource with an element containing only an extension and no value against validation level "Core" will no longer result in an error.
+#. Providing an invalid token to an unsecured operation does not lead to an HTTP 401 error status code.
+
 Feature
 ^^^^^^^
 
-#. Firely Server now fully supports transactions on MongoDb. See :ref:`MongoDb transactions<mongodb_transactions>` in our MongoDB configuration documentation for more details.
+#. Transaction rollbacks are now fully supported when running Firely Server on MogoDB. Please note that the SimulateTransaction setting is now longer available. See :ref:`mongodb_transactions` for more details.
+#. $lastN is now available if Firely Server is running on MongoDB. See :ref:`lastn` for more details.
+#. It is now possible to define exclude filters in the appsettings to configure which requests against Firely Server should not be audited. In certain cases, this enables to reduce the number of captured AuditEvent resources. See :ref:`<feature_auditing>` for more details.
+#. The AuditEvent logging will now by default also capture the the query parameters sent to Firely Server. These parameters will also be stored in case a request fails (HTTP 4xx or 5xx).
```

Comment:
```suggestion
#. By default, the AuditEvent logging will now include the query parameters sent to Firely Server. These parameters will also be stored in case a request fails (HTTP 4xx or 5xx).
```